assets/app/scripts/controllers/overview.js

```diff
@@ -131,9 +239,21 @@ angular.module('openshiftConsole')
           associateDeploymentConfigTriggersToBuild(deploymentConfig, build);
         });
       }
+
+      serviceDeploymentConfigs();      
+
       console.log("deploymentConfigs (subscribe)", $scope.deploymentConfigs);
     }));
 
+    var buildPods = function() {
```

Comment:
remove in favor of filtering monopodsByService to exclude pods we don't ever want to show